packages/core/src/actions/smartAccount/sendTransaction.ts

```diff
@@ -55,8 +62,8 @@ export const sendTransaction: <
   );
   const { hash } = await _sendUserOperation(client, {
     account: account as SmartContractAccount,
-    uoStruct,
+    uoStruct: uoStruct as UserOperationStruct<TEntryPointVersion>,
```

Comment:
tested locally and you don't need this typecast here:
```suggestion
    uoStruct,
```

Reply:
updated

Reply:
bump, still seeing this here?